composer/helpers/src/Hasher.php

```diff
@@ -5,17 +5,12 @@
 namespace Dependabot\Composer;
 
 use Composer\Factory;
+use Composer\Package\Locker;
 
 class Hasher
 {
-    public static function getContentHash(array $args): ?string
+    public static function getContentHash(): string
     {
-        [$workingDirectory] = $args;
```

Comment:
If we need to be able to pass `$workingDirectory` to this method, then we need to make different changes.

Please let me know if that's the case.